fiber/sentryfiber.go

```diff
@@ -51,11 +55,39 @@ func New(options Options) fiber.Handler {
 
 func (h *handler) handle(ctx *fiber.Ctx) error {
 	hub := sentry.CurrentHub().Clone()
-	scope := hub.Scope()
 
-	scope.SetRequest(convert(ctx))
+	if client := hub.Client(); client != nil {
+		client.SetSDKIdentifier(sdkIdentifier)
+	}
+
+	convertedHTTPRequest := convert(ctx)
+
+	options := []sentry.SpanOption{
+		sentry.WithOpName("http.server"),
+		sentry.ContinueFromRequest(convertedHTTPRequest),
+		sentry.WithTransactionSource(sentry.SourceRoute),
+	}
+
+	method := ctx.Method()
+
+	transaction := sentry.StartTransaction(
+		sentry.SetHubOnContext(ctx.Context(), hub),
+		fmt.Sprintf("%s %s", method, ctx.Path()),
+		options...,
+	)
+
+	defer func() {
+		transaction.Status = sentry.HTTPtoSpanStatus(ctx.Response().StatusCode())
+		transaction.Finish()
+	}()
+
+	transaction.SetData("http.request.method", method)
```

Comment:
Any chance we can also intercept the response and set `http.response.status_code`?
https://github.com/getsentry/sentry-go/issues/667